src/resolvers.ts

```diff
@@ -80,10 +84,49 @@ function getStaticChatCompletion(
     return result.choices[0].message.content;
   }
   if (result.choices[0].message.function_call) {
-    return JSON.stringify(result.choices[0].message.function_call);
+    return JSON.stringify({
+      function_call: resolveFunctionCall(
+        result.choices[0].message.function_call,
+      ),
+    });
+  }
+  if (result.choices[0].message.tool_calls) {
+    return JSON.stringify({
+      tool_calls: resolveToolCalls(result.choices[0].message.tool_calls),
+    });
   }
 }
 
+function resolveFunctionCall(
+  call:
+    | ChatCompletionMessageToolCall.Function
+    | ChatCompletionMessage.FunctionCall,
+) {
+  try {
+    return {
+      name: call.name,
```

Comment:
Is this mainly because we are not 100% guaranteed to get valid json? I wonder if we should add an extra flag to the structure when it fails so we (over in prompt-watch) don't have to inspect the arguments field to figure out if it is a string with bad json or not? e.g. error: "invalid_json" or something